pkg/provider/gitlab.go

```diff
@@ -166,7 +171,12 @@ func (repo *GitLabRepository) GetReleases(rawRe string) ([]*semrel.Release, erro
 }
 
 func (repo *GitLabRepository) CreateRelease(release *provider.CreateReleaseConfig) error {
-	tag := fmt.Sprintf("v%s", release.NewVersion)
+	var prefix string
```

Comment:
this should be `prefix := "v"`

Reply:
Absolutely right, it was too late when I did that 😅. Updated.

Reply:
The test suite didn't catch it, I'd like to help refactoring it using mocks for the Gitlab handler but I would do it in another PR if that's ok.

Reply:
Yeah, that would be great ☺️ 🙏